### PR TITLE
serializable nil return

### PR DESCRIPTION
**Pull request by anthonyjlower**
Addresses: https://github.com/LaunchPadLab/lp-serializable/issues/16

### CODE REVIEW

**Comment by anthonyjlower** [comment]
@dpikt I added you as the reviewer since you responded to the issue. Let me know if I should tag someone else.

File: spec/lib/serializable_spec.rb [hunk 1]
```diff
@@ -51,6 +51,11 @@ class UserSerializer
       expect(result).to eq(user_data)
     end
 
+    it "serialize nil, under [:data] if resource is nil" do
```
Comment (dpikt) [inline spec/lib/serializable_spec.rb:54]: Can you add a test case for `serializable_class` as well, even though the behavior hasn't changed?

**Review by dpikt — APPROVED** [review]
Looks good, one non-blocking comment. Also remember to bump the version! (I'd call this a patch.)

**Comment by anthonyjlower** [comment]
@dpikt I added the spec for `serialiazable_class`, but it looks like I don't have permissions to merge PR's on this repo

image

**Comment by dpikt** [comment]
Could you bump the version file too? 🙏 I'll working getting you access

File: spec/lib/serializable_spec.rb [hunk 2]
```diff
@@ -81,6 +86,11 @@ class UserSerializer
       expect(result).to eq(user_data)
     end
 
+    it "Serialize nil, under [:data if resource is nil" do
```
Comment (dpikt) [inline spec/lib/serializable_spec.rb]: Missing closing bracket here 🤓 do you have merge access now?

Comment (anthonyjlower) [reply spec/lib/serializable_spec.rb]: I do have merge access now, thanks!

Comment (dpikt) [reply spec/lib/serializable_spec.rb]: Sweet. Merge at will 👍

**Pull request merged**